docs/topics/secured-mission-springboot-adapter.adoc

```diff
@@ -9,26 +9,25 @@ Both the SSO adapter and endpoint security are configured in `src/main/resources
 [source,bash,options="nowrap",subs="attributes+"]
 ----
 $ # Adapter configuration
-keycloak.realm=master <1>
-keycloak.realm-key=... <2>
-keycloak.auth-server-url=${sso.auth.server.url} <3>
-keycloak.resource=secured-booster-endpoint <4>
-keycloak.credentials.secret=1daa57a2-b60e-468b-a3ac-25bd2dc2eadc <5>
-keycloak.use-resource-role-mappings=true <6>
-keycloak.bearer-only=true <7>
+keycloak.realm=${realm:master} <1>
+keycloak.realm-key=...
+keycloak.auth-server-url=${sso.auth.server.url} <2>
+keycloak.resource=${client.id:secured-booster-endpoint} <3>
+keycloak.credentials.secret=${secret:1daa57a2-b60e-468b-a3ac-25bd2dc2eadc} <4>
+keycloak.use-resource-role-mappings=true <5>
+keycloak.bearer-only=true <5>
```

Comment:
I believe this should be `<6>`.

Reply:
nice catch @tradej 
fixed

Reply:
Thanks, I only discovered it because the validation failed. :)